src/Microsoft.DotNet.Interactive.HttpRequest/extension.dib

```diff
@@ -1,3 +1,7 @@
+#!meta
```

Comment:
The metadata should be left out for this.

Reply:
ok